tests/test_yaml_search_space/inconsistent_types_config2.yml

```diff
@@ -1,18 +1,17 @@
-pipeline_space:
-  learning_rate:
-    type: int
-    lower: 2.3  # float
-    upper: 1e3
-    log: true
+learning_rate:
+  type: int
+  lower: 2.3  # float
+  upper: 1e3
+  log: true
 
-  num_epochs:
-    lower: 3
-    upper: 30
-    is_fidelity: True
+num_epochs:
+  lower: 3
+  upper: 30
+  is_fidelity: True
 
-  optimizer:
-    choices: ["adam", "sgd", "rmsprop"]
+optimizer:
+  choices: ["adam", "sgd", "rmsprop"]
 
-  dropout_rate:
-    value: 0.5
-    is_fidelity: True
+dropout_rate:
+  value: 0.5
+  is_fidelity: True
```

Comment:
Constant fidelity

I might have missed some earlier on but I will discover theses after changing the parameter anywho so no need to address them all